mmdet/datasets/transforms/loading.py

```diff
@@ -819,3 +819,59 @@ def __repr__(self) -> str:
         repr_str += f'with_seg={self.with_seg}, '
         repr_str += f'seg_ignore_label={self.seg_ignore_label})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class InferencerLoader(BaseTransform):
```

Comment:
DetInferncerLoader might be more accurate?

Reply:
I think this class should be placed in mmcv, because most of the repo are exactly the same.